app/activeproject.cpp

```diff
@@ -155,6 +156,10 @@ bool ActiveProject::forceLoad( const QString &filePath, bool force )
     emit projectWillBeReloaded();
     mActiveLayer.resetActiveLayer();
 
+    // import authentication database before loading the layers
```

Comment:
We wanted to reset auth configs before importing new ones..

Reply:
I'd still prefer if we did that so that auth db contains only entries relevant to the opened project and not entries from the previously opened projects.